content/v1.10/reference/troubleshoot.md

```diff
@@ -276,9 +259,9 @@ spec:
 ```
 
 <!-- Named Links -->
-
+<!-- 
```

Comment:
Are these intentionally all commented out?

Reply:
Not sure, maybe an accident during testing. Fixed.